src/Scim/SimpleIdServer.Scim/Commands/Handlers/BaseCommandHandler.cs

```diff
@@ -15,8 +17,20 @@ public BaseCommandHandler(IBusControl busControl)
         {
             _busControl = busControl;
         }
+        
+        protected async Task NotifyAllReferences(List<RepresentationSyncResult> references) {
+            var tasks = new List<Task>();
+            foreach (var reference in references) {
+                // TODO remove
```

Comment:
Why do-you set the Representation property to NULL ?
With the Newtonsoft.Json.JsonIgnore and System.Text.Json.Serialization.JsonIgnore attribute decoration, this property will not be serialized by masstransit.

Reply:
ok cool, I removed this

Reply:
@simpleidserver can you please approve and merge this PR?

Kind regards
Dan